openjpeg.advisories.yaml

```
@@ -0,0 +1,13 @@
+schema-version: "2"
+
+package:
+  name: openjpeg
+
+advisories:
+  - id: CVE-2015-1239
+    events:
+      - timestamp: 2023-09-20T17:48:28Z
+        type: false-positive-determination
+        data:
+          type: vulnerable-code-version-not-used
+          note: 'This was fixed upstream prior to wolfi packaging: https://security-tracker.debian.org/tracker/CVE-2015-1239'
```

The link is helpful! Consider this suggestion just a nit, to help us understand when we're saying it was fixed:

```suggestion
          note: 'This was fixed upstream in version 2.1.1 prior to wolfi packaging: https://security-tracker.debian.org/tracker/CVE-2015-1239'
```